cfme/tests/infrastructure/test_instance_analysis.py

```diff
@@ -208,8 +208,12 @@ def instance(request, local_setup_provider, provider, vm_name, vm_analysis_data)
         image = vm_analysis_data['image']
         vm = VM.factory(vm_name, provider, image)
         request.addfinalizer(vm.delete_from_provider)
-        connect_ip = mgmt_system.get_first_floating_ip()
         provider.refresh_provider_relationships(method='ui')
```

Comment:
A few things here
* We should take the opportunity and refresh via REST I think here
* Secondly, we should perhaps consider adding the assertion of refresh to the method itself